FreeRTOS-Plus/Source/Application-Protocols/network_transport/using_mbedtls_pkcs11/using_mbedtls_pkcs11.c

```diff
@@ -530,12 +526,25 @@ static CK_RV readCertificateIntoContext( SSLContext_t * pSslContext,
     CK_RV xResult = CKR_OK;
     CK_ATTRIBUTE xTemplate = { 0 };
     CK_OBJECT_HANDLE xCertObj = 0;
+    size_t labelLength;
+    char * pcNullTerminator = NULL;
+
+    /* Check for NULL character within pkcs11configMAX_LABEL_LENGTH */
+    pcNullTerminator = memchr(pcLabelName, '\0', pkcs11configMAX_LABEL_LENGTH);
+    if (NULL != pcNullTerminator)
+    {
+        labelLength = pcNullTerminator - pcLabelName;
+    }
+    else
+    {
+        /* If NULL character not found set length to  pkcs11configMAX_LABEL_LENGTH */
+        labelLength = pkcs11configMAX_LABEL_LENGTH;
+    }
 
     /* Get the handle of the certificate. */
     xResult = xFindObjectWithLabelAndClass( pSslContext->xP11Session,
-                                            pcLabelName,
-                                            strnlen( pcLabelName,
```

Comment:
AC6 compiler doesn't have support for `strnlen`. See this issue for why this uses `memchr`: https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/issues/59